Makefile

```diff
@@ -54,7 +54,7 @@ download:
 
 .PHONY: install-tools
 install-tools: download ## Install developer tooling
-	@cd tools && go list -f '{{range .Imports}}{{.}} {{end}}' tools.go | CGO_ENABLED=0 xargs go install -mod=readonly
+	@cd tools && go list -e -f '{{range .Imports}}{{.}} {{end}}' tools.go | CGO_ENABLED=0 xargs go install -mod=readonly
```

Comment:
`-e` is a workaround to prevent failing with:

```
tools.go:7:2: import "github.com/golang/mock/mockgen" is a program, not an importable package
```

https://github.com/golang/go/issues/61857
https://github.com/golang/go/issues/59186